src/main/java/setavailability/SetAvailabilityController.java

```diff
@@ -0,0 +1,23 @@
+package setavailability;
+
+/**
+ * Class responsible for initializing the first-time setting of availability;
+ * takes input of set of LocalTime values & gives it to SetAvailability interactor.
+ *
+ * @author: od-obas1187
+ */
+public class SetAvailabilityController {
+    public static SetAvailabilityInputBoundary userAvailabilityInput;
```

Comment:
Make sure we aren't setting any of the attributes as public; we want to make sure we're encapsulating our fields as much as possible. If an external class really needs to be able to access this data member, which I don't necessarily believe will be the case, have a getter for it.

Furthermore, recall that setting a variable as "static" like this will put it in the static space. I don't believe that having this attribute be static would provide any benefits to your code.